cluster/gce/config-common.sh

```diff
@@ -84,6 +84,20 @@ function get-node-ip-range {
   echo "${suggested_range}"
 }
 
+function get-cluster-ip-range {
```

Comment:
We've had problems using complicated behavior in bash scripts like this before. It makes it hard to stop sourcing these bash scripts

Can we move this dynamic behavior to kubetext or test_context.go?